lib/models/organisation-test.js

```diff
@@ -2,10 +2,71 @@ import chai, { expect } from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { Error } from 'mongoose';
 import Organisation from './organisation';
+import User from './user';
+import Role from './role';
+import { createOrganisation, createUser } from './test-utils/testModelsHelper';
```

Comment:
In case when we will refactor other places or cover another models with tests, we need to create a helper class with util methods for tests. But for now I guess it's ok